src/core/reactor_backend.hh

```diff
@@ -212,6 +212,10 @@ public:
     virtual void start_handling_signal() = 0;
 
     virtual pollable_fd_state_ptr make_pollable_fd_state(file_desc fd, pollable_fd::speculation speculate) = 0;
+
+    virtual bool do_blocking_io() const {
+        return false;
```

Comment:
This should return true or false depending on the detected capabilities.

Reply:
Since it's okay to settle on a 5.6 baseline, it can be unconditionally false.

Reply:
ack.